### DISCUSSION TITLE
Sort columns in specific order

### DISCUSSION THREAD

**Discussion by raffaelle09** [kurkle/chartjs-chart-sankey#91] [General]
Supposed I have a chart with bidirectional data. How do we set the order of the "to" and "from" data in a specific way? 

For this example : https://codepen.io/kurkle/pen/eYvNXqM

How do we make the right side, the same order as the left side? (A,B,C)

**Comment by kurkle** [comment]
You can use `priority`. Updated the pen to have that.

**Reply by raffaelle09** [reply]
Thanks so much! It is working fine now.
